packages/rust/build.sh

```diff
@@ -1,13 +1,12 @@
 TERMUX_PKG_HOMEPAGE=https://www.rust-lang.org/
 TERMUX_PKG_DESCRIPTION="Systems programming language focused on safety, speed and concurrency"
 TERMUX_PKG_LICENSE="MIT"
-TERMUX_PKG_MAINTAINER="Kevin Cotugno @kcotugno"
```

Comment:
I'm removing @kcotugno as maintainer, since he hasn't been doing it for a couple years. No slight, Kevin, but I think it's better if this line were accurate, and we fully understand if you're too busy to volunteer for this. If you'd like me to keep this line the way it is, just let me know.